composer.json

```diff
@@ -59,7 +59,8 @@
     "suggest": {
         "doctrine/data-fixtures": "Load data fixtures"
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
Is there still something that depends on dev?

Reply:
It seems that now, all dependencies are stable.